src/mem/alloc.h

```diff
@@ -516,7 +527,7 @@ namespace snmalloc
       {
         // This is a large alloc redirect.
         ss = ss - (1ULL << (size - 64));
-        size = global_pagemap.get(ss);
+        size = pagemap().get((void*)ss);
```

Comment:
Please don't add C-style casts (they will cause clang-tidy to be unhappy in CI).  You shouldn't need to cast to `void*` (C++ allows implicit casts to `void*`, but not implicit casts *from* `void*`).